community-membership.md

```diff
@@ -157,5 +158,69 @@ The following apply to the subproject(repos) for which one would be an owner:
+Core-Maintainers *MUST* remain active. If they are unresponsive for >3 months, they will be automatically removed unless a super-majority of the other repository maintainers agrees to extend the period to be greater than 3 months.
 
+## Feature-Maintainer
+
+Feature-Maintainers are the technical authority for a subarea of the project/repo in the Dapr org. They *MUST* have demonstrated both good judgement and responsibility towards the health of that subarea. Feature-maintainers can set technical direction and make or approve design decisions for their subarea - either directly or through delegation of these responsibilities.
+
+Defined by: GitHub organization ownership, permissions and entry in `CODEOWNERS`
+files.
+
+### Requirements
+
+The following apply to the sub area(ex: building block API/ Control plane service/ Component provider) for which one would be a Feature-maintainer:
+
+- Deep understanding of the technical goals and direction of the sub area.
+- Deep understanding of the technical domain (specifically the language) of the sub area
+- Sustained contributions to design and direction by doing all of:
+  - Authoring and reviewing proposals
+  - Initiating, contributing and resolving discussions  For example Discord discussions,  GitHub issues, meetings)
+  - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the subarea through implementation and / or review
+- Must have been active for 3 months or more for the given sub-area
+- Inactivity for more than 3 months leads to a removal vote by the repo maintainers and not an automatic removal
+
+
+### Acceptance
+
+  New Feature-Maintainers can be added to the project by a super-majority (two-thirds / 66.66%) vote. Only the maintainers of the repository in which the nomination is applied have a binding vote, while maintainers from other repositories are on an informed basis via a separate email thread.
+
+  A potential feature-maintainer may be nominated by an existing maintainer from the repository in which the nomination is applied to. A vote is conducted in private between the current maintainers over the course of a one week voting period. At the end of the week, votes are counted, and a pull request is made on the repo adding the new feature-maintainer to the feature-maintainers.md file.
+
+  Feature-maintainers MUST remain active. If they are unresponsive for >3 months, they will be automatically removed unless a super-majority of the other repository maintainers agrees to extend the period to be greater than 3 months.
+
+  A feature-maintainer may step down by submitting an issue stating their intent.
+
+### Example of Feature Maintainers in DAPR Runtime repo:
+Sub Area
+Person Name
+Workflow
+Chris Gillum
+Control Plane Service(mTLS)
+Josh V
+
+| **Subarea**   | **Name**                                  |
+| ---------- | ---------------------------------------------| 
+| Workflow    | Chris Gilum                                 | 
```

Comment:
I would use GitHub id here as an example for Chris and Josh

Reply:
Also if these people are treated as maintainers then when voted on, make sure that it is clear that they get maintainers abilities. This means that they should be added to https://github.com/dapr/community/blob/master/MAINTAINERS.md and the sub-area they are responsible for added to this this file. In other words, where do we track who is a feature maintainer and their area?

Reply:
> I would use GitHub id here as an example for Chris and Josh

Agreed, added both name and GH handle.